.goreleaser.release.yml

```diff
@@ -207,8 +207,10 @@ dockers:
     image_templates:
       - 'thethingsnetwork/lorawan-stack:{{ .Major }}.{{ .Minor }}-amd64'
       - 'thethingsnetwork/lorawan-stack:{{ .Version }}-amd64'
+      - 'thethingsnetwork/lorawan-stack:latest-amd64'
```

Comment:
I wonder why the `latest` tag wasn't automatically pushed before. @htdvisser do you recall by any chance ?

Reply:
Because that would result in older versions getting pushed as "latest" if we did backport releases.

Reply:
It would have been nice if we could control whenever or not this tag is pushed via the action parameters. But I don't think that's easy so probably we'll keep the manual push.

Reply:
It is also possible to determine that in the GitHub Actions workflow. If we tag a backport, we'd do so for a previous minor version, so if we just look at the existing tags in the repository, we can only perform that step if we're `>=` the latest tag.